package.json

```diff
@@ -49,6 +49,7 @@
     "cssom": "^0.3.0",
     "domhandler": "^2.3.0",
     "escodegen": "^1.4.1",
+    "eslint": "^1.10.3",
```

Comment:
Is it possible to use something like http://www.webpackbin.com/ or tonicdev to be able to choose versions (might not be directly related to this issue)?